AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -168,42 +166,42 @@ internal class CardViewController: FormViewController {
         
         return address
     }
-
+    
     internal var kcpDetails: KCPDetails? {
         guard
             configuration.koreanAuthenticationMode != .hide,
             let taxNumber = items.additionalAuthCodeItem.nonEmptyValue,
             let password = items.additionalAuthPasswordItem.nonEmptyValue
         else { return nil }
-
+        
         return KCPDetails(taxNumber: taxNumber, password: password)
     }
-
+    
     internal var socialSecurityNumber: String? {
         guard configuration.socialSecurityNumberMode != .hide else { return nil }
         return items.socialSecurityNumberItem.nonEmptyValue
     }
-
+    
     internal var storePayment: Bool? {
         configuration.showsStorePaymentMethodField ? items.storeDetailsItem.value : nil
     }
-
+    
```

Comment:
Why do these line changes happen? Can cause us to miss an actual line change in between

Reply:
I am super keen to figure out why does my swiftformat performs these changes. I'm on the latest Internal repo